src/Domain/Model/PaymentType.php

```diff
@@ -17,4 +18,8 @@
 	const CASH = 'CASH';
 	const BITCOIN = 'BTC';
 
+	public static function getPaymentTypes() {
+		return ( new \ReflectionClass( self::class ) )->getConstants();
```

Comment:
![reflection](https://cloud.githubusercontent.com/assets/146040/13147554/99a9bed2-d659-11e5-8341-8753ed22101f.jpg)

An actually non-evil looking usage of Reflection. Now I've seen it all.

Typical PHP... have `get_class_vars` and `get_class_methods`, but no `get_class_constants`